src/rdb/rdb_raft.c

```diff
@@ -87,31 +87,33 @@ rdb_raft_cb_send_requestvote(raft_server_t *raft, void *arg, raft_node_t *node,
 			     msg_requestvote_t *msg)
 {
 	struct rdb		       *db = arg;
+	rdb_replica_id_t                rdb_node_id;
 	struct rdb_raft_node	       *rdb_node = raft_node_get_udata(node);
 	char			       *s = msg->prevote ? " (prevote)" : "";
 	crt_rpc_t		       *rpc;
 	struct rdb_requestvote_in      *in;
 	int				rc;
 
 	D_ASSERT(db->d_raft == raft);
-	D_DEBUG(DB_TRACE, DF_DB": sending rv%s to node %d rank %u: term=%ld\n",
-		DP_DB(db), s, raft_node_get_id(node), rdb_node->dn_rank,
-		msg->term);
+	D_ASSERT(node != NULL);
+	D_ASSERT(rdb_node != NULL);
+	rdb_node_id = rdb_replica_id_decode(raft_node_get_id(node));
+	D_DEBUG(DB_TRACE, DF_DB ": sending rv%s to node " RDB_F_RID " rank %u: term=%ld\n",
```

Comment:
I suppose rdb_node_id.rri_rank printed as part of the replica ID will be the same value as rdb_node->dn_rank in this log message? Just like before the rank is printed twice?

Reply:
You're right. Duplicated.